src/Tools/Source/RunTests/TestRunner.cs

```diff
@@ -170,6 +169,8 @@ private void PrintFailedTestResult(TestResult testResult)
             ConsoleUtil.WriteLine($"Command: {testResult.CommandLine}");
             ConsoleUtil.WriteLine($"xUnit output log: {outputLogPath}");
 
+            File.WriteAllText(outputLogPath, testResult.StandardOutput ?? "");
```

Comment:
📝 This was moved after console output, so error writing to the file does not prevent writing to the build log.